flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitor.java

```diff
@@ -193,11 +198,23 @@ public WebRuntimeMonitor(
 			this.uploadDir = null;
 		}
 
-		ExecutionGraphHolder currentGraphs = new ExecutionGraphHolder(timeout);
+		final long timeToLive = cfg.getRefreshInterval() * 10L;
```

Comment:
the commit message says it is refreshInterval * 20.

Reply:
yes, because `10 * 2 = 20`.

Reply:
ah right, the message was referring to the cleanupInterval, my bad.